### ISSUE TITLE
[Question]: 合同文本信息抽取效果不好

### ISSUE THREAD

**Issue by zhaoqf-cq** [PaddlePaddle/PaddleNLP#7127]
### 请提出你的问题

需求: 对合同文本进行甲方,乙方,合同编号,合同金额信息抽取

数据标注文件: label_studio.json, 使用 label-studio==1.6.0进行数据标注, 大概选择了38张合同文件, 只标注了甲方,乙方和合同金额, 进行测试

操作平台: aistudio
python默认版本, 好像是3.10.x, paddleocr 和 paddlenlp 都是 pip install --upgrade xxx 安装的
然后从 gitee 拉取的 paddleNlp 的代码, develop 分支的代码
使用PaddleNlp的代码脚本进行 label_studio.json 处理和 模型微调 (本来想尝试 uie-x-base, 但是提示 config.json, model_config.json 文件找不到, 就直接用的 uie-m-large)

操作根据github文档进行模型微调`https://github.com/PaddlePaddle/PaddleNLP/blob/develop/applications/information_extraction/text/README.md`,
微调后使用文档中的进行模型评估, 评估报错了, 我没管, 我直接进行预测, 预测效果不理想, 抽取结果甲方和乙方相同了, 而且金额我标注的都是大写, 如 肆仟元整, 金额出现抽取不出来的情况, 请问应该怎么做才能达到目标效果? 下面是我使用的脚本

数据处理
`python label_studio.py --label_studio_file /home/aistudio/work/label_studio.json --save_dir /home/aistudio/work/data --splits 0.76 0.24 0 --negative_ratio 3 --task_type ext`

模型微调
`python finetune.py  --device gpu --logging_steps 10 --save_steps 100 --eval_steps 100 --seed 1000 --model_name_or_path uie-m-large --output_dir /home/aistudio/work/checkpoint/model_best --train_path /home/aistudio/work/data/train.txt --dev_path /home/aistudio/work/data/dev.txt  --max_seq_len 512  --per_device_train_batch_size  4 --per_device_eval_batch_size 4 --num_train_epochs 20 --learning_rate 1e-5 'end_positions' --do_train --do_eval --do_export --export_model_dir /home/aistudio/work/checkpoint/model_best --overwrite_output_dir --disable_tqdm True --metric_for_best_model eval_f1 --load_best_model_at_end  True --save_total_limit 1
`

模型评估 (这个报错了)
`python evaluate.py --model_path /home/aistudio/work/checkpoint/model_best --test_path /home/aistudio/work/data/dev.txt --device gpu --batch_size 4--max_seq_len 512`

**Comment by xxllp** [comment]
我在1000多数据量下效果还是可以的，或者可以试下用大模型。

**Comment by zhaoqf-cq** [comment]
> 我在1000多数据量下效果还是可以的，或者可以试下用大模型。

我正在尝试ocr的那个大模型... 用paddleX进行训练, 但是一步一个报错,唉... 大佬你抽取的啥啊, 有和我类似的场景么? 模型文件能拿来看看效果么

**Comment by xxllp** [comment]
> > 我在1000多数据量下效果还是可以的，或者可以试下用大模型。
> 
> 我正在尝试ocr的那个大模型... 用paddleX进行训练, 但是一步一个报错,唉... 大佬你抽取的啥啊, 有和我类似的场景么? 模型文件能拿来看看效果么

做的也是合同，pdf转文本然后抽取信息，数据量几千条也不多，类别不是很多。目前是规则为主做的，模型就要自己训练了

**Comment by zhaoqf-cq** [comment]
> > > 我在1000多数据量下效果还是可以的，或者可以试下用大模型。
> > 
> > 
> > 我正在尝试ocr的那个大模型... 用paddleX进行训练, 但是一步一个报错,唉... 大佬你抽取的啥啊, 有和我类似的场景么? 模型文件能拿来看看效果么
> 
> 做的也是合同，pdf转文本然后抽取信息，数据量几千条也不多，类别不是很多。目前是规则为主做的，模型就要自己训练了

我就学习弄着玩儿, 抽取的是合同的甲方, 乙方和合同金额, 我的数都是网上找的, 东一张西一张的, 我试了OCR的直接抽取, 效果不太好, 文本我是先OCR在进行文本标注 然后训练, 不过我样本不多, 而且样本太多样化了, 我是不是应该将 同一个类型的模板多弄几条数据, 然后在进行标注训练, 效果会好一点?

**Comment by yangy996** [comment]
> > > 我在1000多数据量下效果还是可以的，或者可以试下用大模型。
> > 
> > 
> > 我正在尝试ocr的那个大模型... 用paddleX进行训练, 但是一步一个报错,唉... 大佬你抽取的啥啊, 有和我类似的场景么? 模型文件能拿来看看效果么
> 
> 做的也是合同，pdf转文本然后抽取信息，数据量几千条也不多，类别不是很多。目前是规则为主做的，模型就要自己训练了

大佬，我现在也再做合同文本信息提取，但数据集太少了，你那边用规则提取是怎么做的，可以分享一下吗？

**Comment by xxllp** [comment]
> > > > 我在1000多数据量下效果还是可以的，或者可以试下用大模型。
> > > 
> > > 
> > > 我正在尝试ocr的那个大模型... 用paddleX进行训练, 但是一步一个报错,唉... 大佬你抽取的啥啊, 有和我类似的场景么? 模型文件能拿来看看效果么
> > 
> > 
> > 做的也是合同，pdf转文本然后抽取信息，数据量几千条也不多，类别不是很多。目前是规则为主做的，模型就要自己训练了
> 
> 大佬，我现在也再做合同文本信息提取，但数据集太少了，你那边用规则提取是怎么做的，可以分享一下吗？

这个是公司特殊技术，没办法分享~~

**Comment by TenYearsIn** [comment]
麻烦咨询下， 合同文本信息都从哪里获取啊？
